libs/payments/ui/src/lib/client/components/Header/en.tfl

```diff
@@ -0,0 +1,30 @@
+# Component - Header
+
+next-header-help
+  .title = Help
+  .aria-label = Help
+  .alt = Help
+
+next-bento-menu-title =
+  .title = Mozilla products # hover text for bento menu
+  .aria-label = Mozilla products # hover text for bento menu
+  .alt = Mozilla Logo # bento menu logo alt text
+
+next-avatar-menu =
+  .title = Mozilla account menu # hover text for avatar dropdown menu
+  .alt = Account profile picture # profile picture alt text
+
+next-drop-down-menu-signed-in-as-v2 = Signed in as
+next-drop-down-menu-sign-out = Sign out
+
+next-bento-menu-tagline = More products from Mozilla that protect your privacy
+next-bento-menu-close =
+  .alt = Close  # Icon shown in close-menu button
+
+next-bento-menu-firefox-desktop = Firefox Browser for Desktop
+next-bento-menu-firefox-mobile = Firefox Browser for Mobile
+next-bento-menu-monitor-3 = Mozilla Monitor
+next-bento-menu-firefox-relay-2 = Firefox Relay
+next-bento-menu-vpn-2 = Mozilla VPN
+next-bento-menu-pocket-2 = Pocket
+next-bento-menu-made-by-mozilla = Made by Mozilla
```

Comment:
All these string should make use of the branding constant's for the relevant brand. Please see `settings` as an example.

Reply:
Ah, I'll take that as an answer to the response to your other question. I had moved these over directly where possible, and will update them to follow the intended pattern